docs/list.md

```diff
@@ -67,6 +67,14 @@ No one currently! Interested in having a prominent placement for your open sourc
 - **How to sign up**: No special signup known.
 - **Notes**: [Visit Cloudinary](https://cloudinary.com/blog/hacktoberfest-open-source-celebration-cloudinary) for the list of eligible libraries and SDKs.
 
+#### **composio**
+
+- **Swag**: Stickers, T-shirt
+- **Requirements**: Make 4 good PRs against 'hacktoberfest' topics in any of the Composio repositories.
+- **How to sign up**: Register for Hacktoberfest [here](https://hacktoberfest.com/).
```

Comment:
This line is not to register for Hacktoberfest, rather how to register to receive this particular company's swag. Is there a signup for Composio themselves?

Reply:
No they don't have it's a generic link. I guess I can update this to https://hacktoberfest.com/participation/ as how they have it in there HACTOBERFEST_MD file?